### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by alex-golts**
Thank you for publishing this interesting work and very nice code repo!
Please consider merging these few suggested minor changes that I needed in order to successfully run the code:

- removed hardcoded local path
- added a few requirements missing from the conda env file, in a requirements.txt file
- allow to run as script with default arguments
- create a required log dir

### CODE REVIEW

File: train_DTI.py [hunk 1]
```diff
@@ -183,6 +183,7 @@ def main():
     # Logging
     if "log_file" not in config:
         config.log_file = None
+    os.makedirs(os.path.dirname(config.log_file), exist_ok=True)
```
Comment (samsledje) [inline train_DTI.py:186]: Should only do this if log file is specified. L184/L185 above set `config.log_file` to None if not specified, which would throw an error here trying to get the dirname. 

Also, typically would prefer to use `pathlib.Path` for path operations, something like `log_p = Path(config.log_file); os.makedirs(log_p.parent)`

File: train_DTI.py [hunk 2]
```diff
@@ -41,9 +41,9 @@
 
 parser = ArgumentParser(description="PLM_DTI Training.")
 parser.add_argument(
-    "--exp-id", required=True, help="Experiment ID", dest="experiment_id"
+    "--exp-id", help="Experiment ID", dest="experiment_id", default="tmp"
```
Comment (samsledje) [inline train_DTI.py:44]: Okay to make log file default, but would prefer to have experiment ID explicitly set. That way default behavior is not to possibly overwrite previously existing runs

**Review by samsledje — CHANGES_REQUESTED** [review]

**Comment by samsledje** [comment]
Thanks for these helpful fixes Alex! Made a few minor comments above and then okay to merge.

File: train_DTI.py (hunk 1 above)
Comment (alex-golts) [reply train_DTI.py]: done.

File: train_DTI.py (hunk 2 above)
Comment (alex-golts) [reply train_DTI.py]: I reverted this change.
The motivation was to allow running the code as a script for debugging purposes. when "required" is True it's not possible and you have to run it as `python train_DTI.py` and supply CLI arguments.
I understand there are pros and cons to each method and there are also ways to debug with CLI arguments, no big deal.

**Review by samsledje — APPROVED** [review]

**Pull request merged**